### PR TITLE
:bug: Support `raises-exception` in cell execution

### PR DESCRIPTION
**Pull request by agoose77**
This was already supposed to be implemented, but a silly mistake prevented it from working.

### CODE REVIEW

**Comment by agoose77** [comment]
I think the tests are flaking at the moment.

**Comment by choldgraf** [comment]
Looks good, I recommend documenting this as well so that it is clear this is intended behavior

**Comment by agoose77** [comment]
> Looks good, I recommend documenting this as well so that it is clear this is intended behavior

Already on it ;)

File: docs/execute-notebooks.md [hunk 1]
```diff
@@ -28,6 +28,20 @@ The following computational content will be executed:
 In order to execute your MyST content, you must install a Jupyter Server and the kernel needed to execute your code (e.g., the [IPython kernel](https://ipython.readthedocs.io/en/stable/), the [Xeus Python kernel](https://github.com/jupyter-xeus/xeus-python), or the [IRKernel](https://irkernel.github.io/).)
 :::
 
+
+## Deadling with errors
```
Comment (fwkoch) [inline docs/execute-notebooks.md:32]:
```suggestion
## Dealing with errors
```

**Review by fwkoch — APPROVED** [review]
Nice, this makes sense, just a fix to correctly handle the tag.

**Comment by choldgraf** [comment]
I did a quick pass on the docs to make it a bit clearer, I think it's GTG 👍

**Comment by rowanc1** [comment]
Re flaky tests:

image

DOI was down yesterday and the site is still taking a long time on may requests. I think we just wait it out for now.

**Pull request merged**